### PR TITLE
feat: updated pipeline to publish image to docker container

### PR DESCRIPTION
**Pull request by ThisIsDemetrio**
## Pull Request Type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Description

- Updated the `nodejs.yml` to build the docker image (and publishing to DockerHub and Github Registry) after tests when merging Pull Requests to `main` branch;
- Updated the `README.md` file:
  - adding links to the published images;
  - updated description on how to create docker images;
  - included disclaimer on the usage of MongoDB Data Encryption

## PR Checklist

- [X] The commit message follows our guidelines included in the [CONTRIBUTING.md](../CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -44,3 +45,65 @@ jobs:
...
+      - name: Checkout code
+        uses: actions/checkout@v3
+        
+      - name: Configure docker metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ghcr.io/mia-platform/crud-service
+            docker.io/miaplatform/crud-service
```
Comment (danibix95) [inline .github/workflows/nodejs.yml:65]: Is it correct that there is no dash (`-`) here in the label?

File: README.md [hunk 2]
```diff
@@ -280,6 +293,16 @@ The two main communication channel are:
 
 Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for further details about the process for submitting pull requests.
 
+## Disclaimer
+
+### Usage of MongoDB Data Encryption
+
+The CRUD Service can be used along with the [MongoDB Data Encryption](https://www.mongodb.com/basics/mongodb-encryption) functionality, and the images available includes the libraries to use this feature.
+
+However, the MongoDB Data Encryption should be used along with [MongoDB Atlas](https://www.mongodb.com/atlas/database) or [MongoDB Enterprise Advanced](https://www.mongodb.com/products/mongodb-enterprise-advanced). When you use the CRUD Service, for personal projects or commercial applications, you are aware of that and you will comply with [MongoDB terms of use](https://www.mongodb.com/legal/terms-of-use).
```
Comment (danibix95) [inline README.md:302]:
```suggestion
However, the MongoDB Data Encryption should be used along with [MongoDB Atlas](https://www.mongodb.com/atlas/database) or [MongoDB Enterprise Advanced](https://www.mongodb.com/products/mongodb-enterprise-advanced). When you use the CRUD Service, for personal projects or commercial applications, you should be aware of that and you will have to comply with [MongoDB terms of use](https://www.mongodb.com/legal/terms-of-use) accordingly.
```
What do you think about this form?

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (ThisIsDemetrio) [reply .github/workflows/nodejs.yml:65]: Unfortunately, it is.
The organization account is called [miaplatform](https://hub.docker.com/u/miaplatform), so I assume it is correct to use this label.

File: README.md (hunk 2 above)
Comment (ThisIsDemetrio) [reply README.md:302]: I like it more, yeah. I'm going to commit the suggestion as it is.

**Review by danibix95 — APPROVED** [review]
LGTM

**Pull request merged**